rule.go

```diff
@@ -105,6 +106,27 @@ func (cc *Conn) AddRule(r *Rule) *Rule {
 	return r
 }
 
+// DelRule delete the specified rule via its handle number
+// https://wiki.nftables.org/wiki-nftables/index.php/Simple_rule_management#Removing_rules
+func (cc *Conn) DelRule(r *Rule, handle uint64) *Rule {
+
+	data := cc.marshalAttr([]netlink.Attribute{
+		{Type: unix.NFTA_RULE_TABLE, Data: []byte(r.Table.Name + "\x00")},
+		{Type: unix.NFTA_RULE_CHAIN, Data: []byte(r.Chain.Name + "\x00")},
+		{Type: unix.NFTA_RULE_HANDLE, Data: binaryutil.BigEndian.PutUint64(handle)},
```

Comment:
How do you retrieve rule's handle? It is possible that I missed it but I did not see api to get it.

Reply:
Recent changes expose RuleID, Position and Handle information unblocking DelRule..